flake.nix

```diff
@@ -13,13 +14,15 @@
             systems = import systems;
             imports = [ ihp.flakeModules.default ];
 
-            perSystem = { pkgs, ... }: {
+            perSystem = { pkgs, system, ... }: {
                 ihp = {
                     # appName = "app"; # Available with v1.4 or latest master
                     enable = true;
                     projectPath = ./.;
                     packages = with pkgs; [
                         # Native dependencies, e.g. imagemagick
+                        inputs.ghc-wasm-meta.packages.${system}.all_9_10
```

Comment:
IHP is currently pinned to GHC 9.8, which is not an option for the frontend, since there's too much Wasm functionality missing. In fact I believe development of the Wasm backend is mostly targeting 9.12 at this point, with patches being selectively backported to 9.10.

Currently we do get away with using a native GHC 9.8, including for HLS for the frontend, alongside a Wasm GHC 9.10, but this isn't ideal.

Reply:
Ideally we can upgrade IHP to GHC 9.10 soon